core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -566,6 +566,16 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
     }
 
     override def cleanup() {
+      // This is needed to unblock zookeeper listener thread if zookeeper
```

Comment:
Sure. How about this:

"We need to set the iterator to null and notify the rebalance listener thread. This is to handle the case that the consumer rebalance is triggered when the mirror maker thread is shutting down and the rebalance listener is waiting for the offset commit."